homework/HW2_db_table_creator.sql

```diff
@@ -0,0 +1,74 @@
+--- Create database
+DROP DATABASE starschema_company;
+CREATE DATABASE starschema_company ;
+\c starschema_company;
+
+CREATE TABLE order_info (
+  order_number INTEGER PRIMARY KEY,
+  order_date DATE,
+  required_date DATE,
+  shipped_date DATE,
+  status VARCHAR,
+  comments VARCHAR
```

Comment:
Maybe not needed to include in this data warehouse at all, after all, what are you doing to do with the comments?